__tests__/app.test.js

```diff
@@ -101,6 +102,52 @@ describe("GET /api/articles/:article_id", () => {
   });
 });
 
+describe("GET /api/articles/:article_id/comments", () => {
+  test("GET:200 - responds with a 200 status code and an array of comments matching the supplied article id", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ _body: { comments } }) => {
+        expect(comments.length).toBe(2);
+        expect(comments).toBeSorted("created_at",{descending: true });
+        comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.article_id).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+        });
+      });
+  });
+  test("GET:404 - responds with a 404 status code and relevant error message when supplied with a non-existant article_id", () => {
+    return request(app)
+      .get("/api/articles/99/comments")
+      .expect(404)
+      .then(({ _body }) => {
+        expect(_body.msg).toBe("Article with that Id does not exist, nor are there any comments with a matching article Id!");
+      });
+  });
+
+  test("GET:404 - responds with a 404 status code and relevant error message when no comments are found for existing article_id", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
```

Comment:
Would this be a 404? I would suggest here that an article that exists but has no comments is perfectly acceptable. A 200 status code and an empty array coming back should be what is returned to the client here

Reply:
Ha, now that you put it like that, I think I got carried away. Yep that makes sense, I'll reframe and adjust that test and the responses. cheers.